src/lib/containers/SearchV2.tsx

```diff
@@ -1,6 +1,3 @@
-import * as React from 'react'
```

Comment:
There is only one non-whitespace/import change in this file, in the render method